requirements.txt

```diff
@@ -2,6 +2,7 @@
 git+https://github.com/hangoutsbot/hangups.git
 appdirs
 asyncio
+aiohttp >= 0.21.2
```

Comment:
Photo syncs depends on aiohttp, afaik it should be at least aiohttp v0.20.2 in order to download photos and then upload photos with bot APIs